src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -18,6 +18,7 @@ import {
 import { filterPayload } from '../../reducers/util';
 import { addDictionaryReference } from '../bulkConcepts';
 import api from '../../api';
+import instance from '../../../config/axiosConfig';
```

Comment:
What is the meaning of instance?

Reply:
I have used a more appropriate name (`axiosInstance`)